mpisppy/spbase.py

```diff
@@ -97,28 +92,27 @@ def __init__(
             self.branching_factors = self.options["branching_factors"]
         else:
             self.branching_factors = [len(self.all_scenario_names)]
-        self.calculate_scenario_ranks()
-        self.attach_scenario_rank_maps()
+        self._calculate_scenario_ranks()
         if "bundles_per_rank" in self.options and self.options["bundles_per_rank"] > 0:
-            self.assign_bundles()
+            self._assign_bundles()
             self.bundling = True
         else:
             self.bundling = False
-        self.create_scenarios(cb_data)
-        self.look_before_leap_all()
-        self.compute_unconditional_node_probabilities()
-        self.attach_nlens()
-        self.attach_nonant_indexes()
-        self.attach_varid_to_nonant_index()
-        self.create_communicators()
-        self.set_sense()
-        self.set_multistage()
-        self.use_variable_probability_setter()
+        self._create_scenarios(cb_data)
```

Comment:
Where did we land on removing `cb_data`?

Reply:
Yes, I opened an issue to track that (#66). I think we should do it as part of a separate PR.